### ISSUE TITLE
Oracle Linux 8 - iptables error

### ISSUE THREAD

**Issue by tweenietomatoes** [oracle/nosql-examples#26]
Have this program modify firewall settings? (y/n) [n]: y

Setting up firewall rules for ports 5000-5039 on X.X.X.X...
success
Error: COMMAND_FAILED: '/usr/sbin/iptables -w10 -I INPUT_ZONES_SOURCE -p tcp --dport 5000:5039 -s 10.0.0.0/24 -m conntrack --ctstate NEW,ESTABLISHED -j ACCEPT' failed: iptables: No chain/target/match by that name.

Setting up firewall rules for ports 5000-5039 on X.X.X.X...
success
Error: COMMAND_FAILED: '/usr/sbin/iptables -w10 -I INPUT_ZONES_SOURCE -p tcp --dport 5000:5039 -s 10.0.0.0/24 -m conntrack --ctstate NEW,ESTABLISHED -j ACCEPT' failed: iptables: No chain/target/match by that name.

**Comment by tweenietomatoes** [comment]
oracle linux 8 only. 

7 everything working OK.

**Comment by connelly38** [comment]
Modified to work with OEL8.

**Comment by tweenietomatoes** [comment]
1) Actually, i'm also having key-pkcs8.pem issue with proxy on CentOS 8 / Oracle Linux 8.
It's not happening on CentOS / OCL 7.
Proxy is not taking key-pkcs8.pem file. Giving error in httpproxy.out **as your key-pkcs8.pem is not valid.** , i'm not sure if it's cluster_setup.sh 's problem. I also recreated key myself and problem stays. So it seems like proxy's problem. Is CentOS / OCL 8 creating key-pkcs8 with different format?

2) Having issue with makebootconfig on Debian JDK, Ubuntu JDK, AdoptOpenJDK.
Tried passing -force option in cluster_setup. Nothing changed.
